package.json

```diff
@@ -24,55 +24,55 @@
   ],
   "scripts": {
     "build": "webpack --progress --config resources/assets/build/webpack.config.js",
-    "build:production": "webpack --env.production --progress --config resources/assets/build/webpack.config.js",
+    "build:production": "webpack --mode=production --progress --config resources/assets/build/webpack.config.js",
     "build:profile": "webpack --progress --profile --json --config resources/assets/build/webpack.config.js",
-    "start": "webpack --hide-modules --watch --config resources/assets/build/webpack.config.js",
+    "start": "webpack --watch --config resources/assets/build/webpack.config.js",
     "rmdist": "rimraf dist",
     "lint": "npm run -s lint:scripts && npm run -s lint:styles",
     "lint:scripts": "eslint resources/assets/scripts resources/assets/build",
     "lint:styles": "stylelint \"resources/assets/styles/**/*.{css,sass,scss,sss,less}\"",
     "test": "npm run -s lint"
   },
   "engines": {
-    "node": ">= 8.0.0"
+    "node": ">= 12.18"
   },
   "devDependencies": {
     "autoprefixer": "^10.0.1",
     "browser-sync": "^2.26.13",
-    "browsersync-webpack-plugin": "^0.6.0",
+    "browser-sync-webpack-plugin": "^2.3.0",
     "bs-html-injector": "~3.0",
-    "buble-loader": "^0.4.1",
-    "cache-loader": "~1.2.5",
-    "clean-webpack-plugin": "^0.1.18",
-    "copy-globs-webpack-plugin": "^0.2.0",
-    "css-loader": "^0.28.11",
+    "buble": "^0.20.0",
+    "buble-loader": "^0.5.1",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^7.0.0",
```

Comment:
Hey guys. thanks for all the work during this PR!

It seems that copy-webpack-plugin v7.0.0 was removed for some reasons and v8.0.0 already released with some breaking changes: https://github.com/webpack-contrib/copy-webpack-plugin/releases/tag/v8.0.0

Reply:
https://github.com/roots/sage/pull/2607/commits/37c000596314184002d6ccc3afdb90da8abbce29